app/models/room.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+#
+# Model to represent the lowest-level of facility information
+#
+# Attributes:
+#   - number: The number of the building, a string
+#     Required, must be unique within the suite the room belongs to
+#
+class Room < ApplicationRecord
+  belongs_to :suite
```

Comment:
`belongs_to through` associations aren't a thing -- it looks like we can use `delegate` to get similar behavior? (see this [stack overflow](http://stackoverflow.com/a/11457714)) I don't think it's an ideal solution, though...